third_party/build/build-dist.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+# Build a stubs-only distribution for a third-party package.
+#
+# Usage: python3.7 build-dist.py <PACKAGE>
```

Comment:
I'm not sure requiring people to have a 3.7 install is a good idea, but then again it doesn't bother me too much. At minimum a version check/warning before importing anything would avoid people being confused by "no module named dataclasses".

Reply:
Eventually this script is meant to be run as part of the CI process, do I didn't worry about that too much. But the version check is a good idea!